stable/datadog/values.yaml

```diff
@@ -401,10 +401,16 @@ kube-state-metrics:
     #
     create: true
 
-    ## @param serviceAccountName - string - required
-    ## Ignored if rbac.create is true
-    #
-    serviceAccountName: default
```

Comment:
Why removing this ? we are using `.Values.rbac.serviceAccountName` not `.Values.rbac.serviceAccount.name`
in the different manifests.
For instance: https://github.com/helm/charts/blob/master/stable/datadog/templates/cluster-agent-deployment.yaml#L164

Reply:
if you look the full path of this variable it is: `kube-state-metrics.rbac.serviceAccountName`  and not `.Values.rbac.create` as we use in your link.

In fact in `kube-state-metrics` chart, they replace this variable by `.Value.rbac.serviceAccount.name`. That is why I added the following fields